### ISSUE TITLE
Docs: Ensure demo pages include required props

### ISSUE THREAD

**Issue by paulcpederson** [Esri/calcite-components#1576]
### Description 

Some of the demos are missing props that are required but the component. Below is an exhaustive list of props declared as required in our code:

- calcite-action
  - [x] text
- calcite-alert
  - [x] label
- calcite-chip
  - [x] value
- calcite-combobox
  - [x] label
- calcite-combobox-item
  - [x] textLabel
  - [x] value
- calcite-loader
  - [x] label
- calcite-option-group
  label
- calcite-pick-list-item
  - [x] value
- calcite-popover
  - [x] label
  - [x] referenceElement
- calcite-radio-button
  - [x] value
- calcite-radio-button-group
  - [x] name
- calcite-select
  - [x] label
- calcite-tooltip
  - [x] label
  - [x] referenceElement
- calcite-tree-item
  - [x] childrenSlotWrapper
  - [x] defaultSlotWrapper
- calcite-value-list-item
  - [x] label
  - [x] value

For each component listed above, we need to look at their demo page (or search for the component name in `demos/` folder) and validate that every occurrence of the tag includes these props. 

### Example

Say you're checking the loader component, you'd open `src/demos/calcite-loader.html`, check all the instances of the `<calcite-loader>` element. As you can see, [some don't have the required loader prop](https://github.com/Esri/calcite-components/blob/master/src/demos/calcite-loader.html#L84). Edit the file adding the required prop:

```diff
- <calcite-loader active inline></calcite-loader><span style="margin:0 10px">Inline Loader</span>
+ <calcite-loader label="loading" active inline></calcite-loader><span style="margin:0 10px">Inline Loader</span> 
```

These props are generally labels for `aria` (screen readers), but if you're not sure what they do, you can check the readme for each component and the prop should have a description in the props table.

**Note**: you may need to also search the demos folder for other instances of `loader` incase _other_ components use the loader in _their_ demos. For example, the loader element [also appears on the index page](https://github.com/Esri/calcite-components/blob/dd10e9c2e2035715b1b72b87f90c6b86ef7f56fd/src/index.html#L378)
